src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -437,7 +437,7 @@ private function getShell()
      */
     private function readFromInput($stream)
     {
-        if (STDIN === $stream && function_exists('readline')) {
+        if (STDIN === $stream && function_exists('readline') && !defined('HHVM_VERSION')) {
             $ret = readline();
```

Comment:
giving the empty string looks more feature-full, isn't it? why prefer fgets on hhvm when readline works?